config/whitelabel.yml

```diff
@@ -142,19 +142,22 @@
   - gogolok
   location:
     :zoom: 13
-    :lat: 49.23856
-    :long: 6.996574
+    :lat: 49.2429927
+    :long: 6.9897586
   default_time_zone:
     "Europe/Berlin"
   imprint:
     :address: |-
-      Science Park 2
-      66123 Saarbrücken
+      Ligatura
+      Europaallee 27
+      66113 Saarbrücken
     :contributors:
     - :name: Markus Schwed
       :email: ma.schwed@googlemail.com
     - :name: Robert Gogolok
       :email: gogolok@gmail.com
+    - :name: CompanyMood
+      :email: https://www.company-mood.de
```

Comment:
Ok, got the problem.  It get linked via `mailto:`  

It's been a while, when i added those lines for Avarteq. 
Give me a sec, I'll take both out. :+1: